tracer/src/Datadog.Trace/OpenTelemetry/Logs/OtlpExporter.cs

```diff
@@ -0,0 +1,261 @@
+            }
+            else
+            {
+                var opt = new OtlpExporterOptions
+                {
+                    Endpoint = _endpoint,
+                    TimeoutMilliseconds = _timeoutMs,
+                    Protocol = (OtlpExportProtocol)_protocol,
+                    AppendSignalPathToEndpoint = false // HTTP endpoint already includes /v1/logs
+                };
+
+                foreach (var kvp in _headers)
+                {
+                    opt.Headers[kvp.Key] = kvp.Value;
+                }
+
+                const string logsHttpPath = "v1/logs";
+                _httpExportClient = new OtlpHttpExportClient(opt, _httpClient, logsHttpPath);
+            }
+        }
+
+        /// <summary>
+        /// Exports a batch of logs using OTLP protocol.
+        /// This method implements the LogExporter base class Export operation.
+        /// </summary>
+        /// <param name="logs">Batch of logs to export</param>
+        /// <returns>ExportResult indicating success or failure</returns>
+        public ExportResult Export(IReadOnlyList<LogPoint> logs)
+        {
+            // For backward compatibility, call the async version synchronously
+            return ExportAsync(logs).GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// Exports a batch of logs using OTLP protocol asynchronously.
+        /// This is the preferred method for better performance.
+        /// </summary>
+        /// <param name="logs">Batch of logs to export</param>
+        /// <returns>ExportResult indicating success or failure</returns>
+        public async Task<ExportResult> ExportAsync(IReadOnlyList<LogPoint> logs)
+        {
+            if (logs.Count == 0)
+            {
+                Log.Debug("No logs to export.");
+                return ExportResult.Success;
+            }
+
+            try
+            {
+                var success = await SendOtlpRequest(logs).ConfigureAwait(false);
+                return success ? ExportResult.Success : ExportResult.Failure;
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error exporting OTLP logs.");
+                return ExportResult.Failure;
+            }
+        }
+
+        /// <summary>
+        /// Shuts down the exporter and ensures all pending exports complete.
+        /// </summary>
+        /// <param name="timeoutMilliseconds">Maximum time to wait for shutdown</param>
+        /// <returns>True if shutdown completed successfully</returns>
+        public bool Shutdown(int timeoutMilliseconds)
+        {
+            try
+            {
+                _httpClient.CancelPendingRequests();
+                _httpClient.Dispose();
+                Log.Debug("OTLP Log Exporter shutdown completed.");
+                return true;
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error during OTLP Log Exporter shutdown.");
```

Comment:
I'm guessing `CancelPendingRequests` may throw an exception. In which case, we may want to catch those exceptions explicitly, or otherwise downgrade this to warning. Unless we never _Should_ have ongoing requests, in which case we're probably fine